frontend/src/store/uam/actions.ts

```diff
@@ -0,0 +1,17 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { AsyncThunkConfig, UsersGetAllResponseDto } from 'common/types/types';
```

Comment:
Seems like you're waiting for `UsersGetAllResponseDto` but returning an array from BE, same in reducer - you've declaring users as array, but type them in store as `UsersGetAllResponseDto`